### DISCUSSION TITLE
Alignment using surfaces

### DISCUSSION THREAD

**Discussion by ghost** [OrcaSlicer/OrcaSlicer#5149] [Ideas]
It would be great if there was a way to align objects based on the flat surfaces. Using the same surface selector you can align an object to the print bed, it would be sweet if you could select a second surface on another object and align the two object together based on that. Like if you wanted to quickly connect to objects. Additionally, if you could rotate an object around an axis perpendicular to the surface you selected, you could then quickly align two objects and line them up how you want within an assembly for printing together.

**Comment by vgdh** [comment]
It would be great!

**Reply by MagPi-314** [reply]
I have just found this thread whilst trying to do the very same thing.  I CTRL selected the two models, one nicely aligned on the bed and the other at a quirky angle. I then clicked the Assemble icon, first face you click on is the fixed (correct orientation) then click a face on the other model and it will align. They remain separate items. Does this help? It helped me 😁
